packages/zk-transfer-web/src/App.tsx

```diff
@@ -13,7 +14,15 @@ type AppProp = {
 };
 
 function App({ demo } : AppProp) {
-  // await api.init(demo);
+  const [chainId, setDemoChain] = useState(demo.chainId);
+
+  useEffect(() => {
+    if (_.isUndefined(chainId))
+      demo.clientInit()
+        // .then(() => { console.log(chainId) })
+        .then(() => setDemoChain(demo.chainId));
```

Comment:
and add a closing brace here please, to keep consistent style
```
  }
```

Reply:
Ok.